PodcastGenerator/core/feed_generator.php

```diff
@@ -116,6 +117,19 @@ function generateRSS()
         $item = '
 		<item>
 			<title><![CDATA[' . $file->episode->titlePG . ']]></title>
+			';
+
+        if ($file->episode->episodePG == '') {
+            $item .= '<itunes:episode>' . $file->episode->episodePG . '</itunes:episode>
+            ';
+        }
+        if ($file->episode->seasonPG == '') {
+            $item .= '<itunes:season>' . $file->episode->seasonPG . '</itunes:season>';
+        }
```

Comment:
Thanks a lot for this feature but shouldn't we put this into the episode upload dialog as well? And what is with old episodes? I would suggest adding this to the backwards compatibility converter